addons/event/tests/test_event_flow.py

```diff
@@ -1,18 +1,102 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-import datetime
+from datetime import datetime, timedelta
 
 from dateutil.relativedelta import relativedelta
 
+from odoo import Command
 from odoo.addons.event.tests.common import TestEventCommon
 from odoo.exceptions import ValidationError
+from odoo.fields import Datetime
 from odoo.tests.common import Form
 from odoo.tools import mute_logger
 
 
 class TestEventUI(TestEventCommon):
 
+    def test_event_event_type_id_edition(self):
```

Comment:
Actually there are already some tests from template to event propagation (see test_event_configuration_from_type in test_event_internals). Could be great to move them either in that test, either move and split the test (one test for tickets, one for mails, one for "small fields"). That way test_event_internals covers all "corner cases" and test_event_flow stays a bit more high-level.